config/__tests__/__snapshots__/config.test.test.js.snap

```diff
@@ -0,0 +1,111 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`test configuration should load correct configuration: config-keys 1`] = `
+Array [
+  "configTag",
+  "authsome.isPublic",
+  "authsome.mode",
+  "validations",
+  "pubsweet.components.0",
+  "pubsweet.components.1",
+  "pubsweet.components.2",
+  "pubsweet.components.3",
+  "pubsweet.components.4",
+  "pubsweet.components.5",
+  "pubsweet.components.6",
+  "pubsweet.components.7",
+  "pubsweet.components.8",
+  "pubsweet.components.9",
+  "pubsweet.components.10",
+  "pubsweet.components.11",
+  "pubsweet.components.12",
+  "pubsweet.components.13",
```

Comment:
if you really want to maintain these lists everytime a configuration key changes, up to you

Reply:
We can alter this test as we go ... at least we have one now :)

Reply:
done - please check!